### ISSUE TITLE
Not all historical data available via api?

### ISSUE THREAD

**Issue by shrippen** [robvankeilegom/firefly-III-paypal-importer#21]
I have another question. I had a look in the database. The importer fetches all transactions up until 2021-02-01. Then there is a gap but some further transcations are picked up for Mai 2019 and a bit of November/December 2017. If I export a CSV on the paypal website I can get all data with no gaps in the same time range. Do you have any idea why?

**Comment by robvankeilegom** [comment]
Hey! Sorry for the late response. The application stopped syncing when it reached an empty month. I fixed this in `develop`. You can now pass `--keep-going` to the `sync` command. So you should run this once manually. If anything isn't clear, lmk!

**Comment by shrippen** [comment]
Thanks for your time. Unfortuantely, when I try to execute the sync command with --keep-going artisan tells me that this option does not exist. I am running this in a docker container and updated to the 0.3.4 version.

![image](https://github.com/robvankeilegom/firefly-III-paypal-importer/assets/2873570/41f4d38a-e9ec-4e22-9d1c-3f4a13bbcb60)

**Comment by robvankeilegom** [comment]
Hey, i'm sorry. this is in the `develop` tag. This hasn't made it into a release yet.

**Comment by ryanjkemper** [comment]
I'm having the same issue using the `latest` tag in docker, and I don't notice any new options related to the new flag int he `develop` branch in git.  Can you confirm if you have these changes available for testing?

**Comment by robvankeilegom** [comment]
@ryanjkemper Yeah it's in develop. You can find the changes here https://github.com/robvankeilegom/firefly-III-paypal-importer/blob/fe4e0bd3a118484b74e37b7384d8377a109702d4/app/Console/Commands/Sync.php#L15C42-L15C47

**Comment by LnLcFlx** [comment]
Is the command supposed to not exit when running with `--keep-going`?
For me, it does not exit after having reached `7/2015` and waiting for 5min.
